docs/sql-schemas/payments_sql_schema.mermaid

```diff
@@ -0,0 +1,125 @@
+erDiagram
```

Comment:
You can also make this a markdown file, then wrap this in:
```
```mermaid
```
```

That'll allow users to blow it up to full screen.